CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Add the fields to retrieve when loading the data from argilla. `rg.load` takes too long because of the vector field, even when users don't need it. Closes [#2398](https://github.com/argilla-io/argilla/issues/2398)
 - Add new page and components for dataset settings. Closes [#2442](https://github.com/argilla-io/argilla/issues/2003)
+- Accept non-searchable fields in metadata. See [#2570](https://github.com/argilla-io/argilla/pull/2570)
```

Comment:
Some suggestions:
- Write these lines like imaging that they start with `Added`.
- I believe you don't need to add the URL for the issue, more info about this [here](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/autolinked-references-and-urls#issues-and-pull-requests).
- Let's use the standard referencing issues.

```suggestion
- Non-searchable fields in metadata. (#2570)
```